packages/babel-parser/test/fixtures/es2017/async-functions/no-method-asi/output.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
oh I would of expected these tests to be duplicated given they used to throw with the enable parser plugin error (and thus wouldn't be testing the syntax but the plugin), but I guess they are unique actually? Like we would of had a test for asi that didn't throw as well, so you could delete this one.